lab_1/main.py

```diff
@@ -21,7 +36,17 @@ def remove_stop_words(tokens: list, stop_words: list) -> list or None:
     :param stop_words: a list of stop words
     :return: a list of tokens without stop words
     """
-    pass
+    if isinstance(tokens, list) and isinstance(stop_words, list):
+        if tokens:
+            for token in enumerate(tokens):
+                if token[1] in stop_words:
+                    tokens[token[0]] = "@@@"
+            while "@@@" in tokens:
```

Comment:
what are these symbols? can you avoid having these symbols by using regular expressions?